fmn/rules/taskotron.py

```diff
@@ -10,7 +10,8 @@ def taskotron_result_new(config, message):
     """
     return message['topic'].endswith('taskotron.result.new')
 
-@hint(categories=['taskotron'], invertible=False)
+
+@hint(categories=['taskotron'])
```

Comment:
So _invertible_ is only related to _hint_ and only affects showing the examples to the user, but it does not prevent the user from negating the rule itself, right?

In that case, sounds logical to me to have `invertible=False` for all the rules in this patch.